File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/alerts")
    print(f"Title of the page: {driver.title}")

    driver.find_element(By.ID, "prompt").click()
    prompt_alert = driver.switch_to.alert
    print("Text from prompt alert: " +prompt_alert.text)
		
    prompt_alert.send_keys("Awesome!")
    prompt_alert.accept()
		
    result = driver.find_element(By.ID, "result").text
		
    print("Result: " +result)

File: Selenium/Activities/Activity20.java
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity20 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/alerts");
		System.out.println("Title of the page: " +driver.getTitle());
		
		WebElement promptAlertButton = driver.findElement(By.id("prompt"));
		promptAlertButton.click();
		Alert promptAlert = driver.switchTo().alert();
		System.out.println("Text from prompt alert: " +promptAlert.getText());
		
		promptAlert.sendKeys("Awesome!");
		promptAlert.accept();
		String result = driver.findElement(By.id("result")).getText();
		System.out.println("Result: " +result);
		
		driver.quit();		
	}
}
